### PR TITLE
equations

### PR DESCRIPTION
**Pull request by MikeLippincott**
Figure that shows equations used to get the differential features across comparisons

### CODE REVIEW

File: figures/eq1/figures/equations4_9.png
[the remark this answers is not in the document]
Comment (MattsonCam) [reply figures/eq1/figures/equations4_9.png:1]: Are N the number of rows, and (P and Q) the number of columns? Consider specifying this as R^{nxm} or something like that.

[the remark this answers is not in the document]
Comment (MattsonCam) [reply figures/eq1/figures/equations4_9.png:1]: May also indikate that a^hat and b^hat come from A^hat and B^hat if that's the case

[the remark this answers is not in the document]
Comment (MattsonCam) [reply figures/eq1/figures/equations4_9.png:1]: Is rk^2 the coefficient of determination here? Also, what were the inputs to calculate it?

**Review by MattsonCam — APPROVED** [review]
Hey @MikeLippincott , just added some suggestions, good job! Maybe you explain some of the parts my suggestions reference somewhere else, so I will approve.

File: figures/eq1/figures/equations4_9.png
[the remark this answers is not in the document]
Comment (MikeLippincott) [reply figures/eq1/figures/equations4_9.png:1]: I addressed each of these comments and am ready for a re-review!

[the remark this answers is not in the document]
Comment (MattsonCam) [reply figures/eq1/figures/equations4_9.png:1]: Is a^{squiggly}_{pk} = A^{squiggly}_{pk}? Sorry, I don't know what the squiggly is called above the variable

[the remark this answers is not in the document]
Comment (MattsonCam) [reply figures/eq1/figures/equations4_9.png:1]: > Are N the number of rows, and (P and Q) the number of columns? Consider specifying this as R^{nxm} or something like that.

I think the way you did it is the same

[the remark this answers is not in the document]
Comment (MattsonCam) [reply figures/eq1/figures/equations4_9.png:1]: Do the lowercase variables come from the uppercase variables?

**Review by MattsonCam — APPROVED** [review]
This looks much better, good job @MikeLippincott!

File: figures/eq1/figures/equations4_9.png
[the remark this answers is not in the document]
Comment (MikeLippincott) [reply figures/eq1/figures/equations4_9.png:1]: Yes I am not sure how to convey this properly. But the lowercase variables are the uppercase variables at a given index (i,j)

**Pull request merged**